rpd_generator/bdl_structure/bdl_commands/circulation_loop.py

```diff
@@ -148,15 +148,14 @@ def populate_data_elements(self):
             self.type = self.loop_type_map.get(loop_type, FluidLoopOptions.OTHER)
 
             # Populate the data elements for FluidLoopDesignAndControl
-            if self.circulation_loop_type in ["FluidLoop", "SecondaryFluidLoop"]:
```

Comment:
I think we need this line

Reply:
We do the same check right before in line 146 above. The conditional at line 151 will always be true

Reply:
I see, thanks